packages/user/src/views/SignUpFirstUser.tsx

```diff
@@ -0,0 +1,118 @@
+import { useTranslation } from "@dzangolab/react-i18n";
+import { Page } from "@dzangolab/react-ui";
+import { Card } from "primereact/card";
+import { useEffect, useState } from "react";
+import { toast } from "react-toastify";
+
+import { getIsFirstUser, signUpFirstUser } from "@/api/user";
+import SignupForm from "@/components/SignupForm";
+import { useConfig, useUser } from "@/hooks";
+
+import { login } from "..";
+
+import type { LoginCredentials } from "@/types";
+
+const SignUpFirstUser = () => {
+  const { t } = useTranslation("user");
+
+  const appConfig = useConfig();
+  const { setUser } = useUser();
+
+  const [loading, setLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isFirstUser, setIsFirstUser] = useState<boolean>(false);
+
+  const [signUpFirstUserLoading, setSignUpFirstUserLoading] = useState(false);
+  const [loginLoading, setLoginLoading] = useState(false);
+
+  useEffect(() => {
+    getIsFirstUser(appConfig?.apiBaseUrl || "")
+      .then((response) => {
+        setIsFirstUser(response.signUp);
+      })
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, []);
+
+  const handleSubmit = (credentials: LoginCredentials) => {
+    if (!isFirstUser) {
+      return;
+    }
+
+    setSignUpFirstUserLoading(true);
+
+    signUpFirstUser(credentials, appConfig?.apiBaseUrl || "")
+      .then((response) => {
+        setSignUpFirstUserLoading(false);
+        setLoginLoading(true);
+
```

Comment:
Toast a success message here for "Succefully added first admin user. Automatically logging in ..." message.